examples/event-handling-using-read-models/InvitationProjectorsTest.php

```diff
@@ -28,7 +28,7 @@ protected function createProjector(Broadway\ReadModel\InMemory\InMemoryRepositor
     /**
      * @test
      */
-    public function it_keeps_track_of_the_status_of_an_invitation_when_someone_is_invited()
+    public function itKeepsTrackOfTheStatusOfAnInvitationWhenSomeoneIsInvited()
```

Comment:
can you remove this commit? It has nothing to do with PHP8 and also we like the original casing.

Reply:
Code quality tests don't pass without this commit.

Reply:
Hmm, why did they pass before?

Reply:
I think that with the new versions of PHPCsFixer they have changed the behavior.

Reply:
@othillo to resolve the problem we can use this configuration for tests to force using snake case

```
'php_unit_method_casing' => ['case' => 'snake_case']
```

What do you think?

Reply:
if that would help to remove this commit, I'm +1!

Reply:
@othillo I have added to the configuration that rule and I have rebased the branch.